superbench/benchmarks/micro_benchmarks/gpu_copy_bw_performance.py

```diff
@@ -78,6 +84,9 @@ def _preprocess(self):
         for copy_type in self._args.copy_type:
             args += ' --%s_copy' % copy_type
 
+        if self._args.bidirectional:
+            args += ' --bidirectional'
```

Comment:
do we need space?

Reply:
Yes the args is appended directly to the real executing command.